forge/csrc/passes/commute_utils.cpp

```diff
@@ -45,9 +45,12 @@ std::tuple<bool, int> can_commute_reshape_through_dim(
     {
         if (input_shape_vec[i] == output_shape_vec[dim])
         {
+            log_info("output_shape_vec.size() = {}", output_shape_vec.size());
```

Comment:
As we're adding these logs, it'll be good to add a few more for easier debugging of this pass. Also, these are better fit for trace logs